systems/plants/TimeSteppingRigidBodyManipulator.m

```diff
@@ -17,6 +17,7 @@
     enable_fastqp; % whether we use the active set LCP
     lcmgl_contact_forces_scale = 0;  % <=0 implies no lcmgl
     z_inactive_guess_tol = .01;
+    mex_model_ptr = 0; % needed for access from mex functions
```

Comment:
Ah, true, this is no longer strictly necessary. I had done this to enable me to call mex functions and matlab functions with the same signature, but that's no longer an important part of my approach, so I'll remove this.